models/articles.models.js

```diff
@@ -57,15 +60,22 @@ exports.insertCommentByArticleId = (article_id, comment) => {
     })
 }
 
-exports.updateArticleById = (article_id,body) => {
-  const {inc_votes} = body
-  if (!inc_votes || (typeof inc_votes !== 'number')) {
-    return Promise.reject({status:400,customErrMsg:'invalid vote increment supplied'})
+exports.updateArticleById = (article_id, body) => {
+  const { inc_votes } = body
+  if (!inc_votes || typeof inc_votes !== 'number') {
```

Comment:
I'm pretty sure if you try to do either of these things you should get a psql error code to save you a custom promise reject. However, I think computationally, stopping it at this point is a little more efficient than letting the database kick the query back